lib/moab/exceptions.rb

```diff
@@ -1,13 +1,9 @@
 module Moab
-  class ObjectNotFoundException < RuntimeError
-  end
+  class MoabRuntimeError < RuntimeError; end
```

Comment:
I wonder whether you're open to declaring a single `MoabError` class rather than two classes. Or maybe there is some value to distinguishing between a Moab "standard" error and a Moab "runtime" error? That was there is a single class for all custom errors in this codebase.

Reply:
The value is that one of the exceptions that used to be thrown is StandardError, so it seemed useful to do it this way.  My goal is to do a minor release of MoabVersion so there is absolutely no ill effect to any app using the gem -- I don't want to cause work for any consumers except pres_cat, where John and Peter did a kludge because they couldn't do this PR at the time.

Reply:
@ndushay Hmm, I'm not sure I follow. Are you saying that the value of making subclasses of StandardError and RuntimeError is that downstream code can rescue the `RuntimeError` children without accidentally rescuing the `StandardError` children?

Reply:
yup - just in case.